packages/ui/src/components/ChatMessage/ChatMessageActions/Component.tsx

```diff
@@ -93,6 +98,10 @@ export function ChatMessageActions({
   }
 
   useEffect(() => {
+    if (isWeb) {
+      // On web we don't have haptics
```

Comment:
I think we have the `Haptics` module set up to no-op on Web. Would be nice to avoid having to think about platform whenever calling it if possible

Reply:
Looks like we were just importing the Haptics module from expo-haptics directly in a few places, fixed that in the latest commit.